client/components/Assignments/AssignmentItem/fields/index.ts

```diff
@@ -0,0 +1,55 @@
+import {AcceptedComponent} from './Accepted';
+import {ContentComponent} from './Content';
+import {DescriptionTextComponent} from './DescriptionText';
+import {DeskComponent} from './Desk';
+import {DueDateComponent} from './DueDate';
+import {GenreComponent} from './Genre';
+import {HeadlineComponent} from './Headline';
+import {InternalComponent} from './Internal';
+import {NameComponent} from './Name';
+import {PriorityComponent} from './Priority';
+import {SluglineComponent} from './Slugline';
+import {StateComponent} from './State';
+
+export const getComponentForField = (field: string) => {
+    switch (field) {
+    case 'accepted':
+        return AcceptedComponent;
+    case 'content':
+        return ContentComponent;
+    case 'description_text':
+        return DescriptionTextComponent;
+    case 'desk':
+        return DeskComponent;
+    case 'due_date':
+        return DueDateComponent;
+    case 'genre':
+        return GenreComponent;
+    case 'headline':
+        return HeadlineComponent;
+    case 'internal':
+        return InternalComponent;
+    case 'name':
+        return NameComponent;
+    case 'priority':
+        return PriorityComponent;
+    case 'slugline':
+        return SluglineComponent;
+    case 'state':
+        return StateComponent;
+    }
+};
+
+export const DEFAULT_ASSSIGNMENTS_LIST_VIEW = {
+    firstLine: ['slugline', 'description_text', 'name'],
```

Comment:
In the current codebase, only `slugline` and `description_text` are displayed, the default should reflect this.

Reply:
Yeah I left it there by mistake, I'll change it back